boa3_test/tests/compiler_tests/test_list.py

```diff
@@ -629,7 +643,166 @@ def test_list_slicing_end_omitted(self):
+        )
+
+        expected_negative_lower_index_output = (
+            Opcode.INITSLOT     # negative_start function signature
+            + b'\x01'
+            + b'\x00'
+            + Opcode.PUSH5
+            + Opcode.PUSH4
+            + Opcode.PUSH3
+            + Opcode.PUSH2
+            + Opcode.PUSH1
+            + Opcode.PUSH0
+            + Opcode.PUSH6
+            + Opcode.PACK       # [0, 1, 2, 3, 4, 5]
+            + Opcode.STLOC0
+            + Opcode.LDLOC0
+            + Opcode.PUSH6
+            + Opcode.NEGATE
+
+            + fix_negative_lower_index
+            + check_lower_index_out_of_bounds
+
+            + Opcode.PUSH5
+            + check_upper_index_out_of_bounds
+            + Opcode.NEWARRAY0  # starts getting the subarray
+        )
+        expected_negative_upper_index_output = (
+            Opcode.INITSLOT     # negative_end function signature
+            + b'\x01'
+            + b'\x00'
+            + Opcode.PUSH5
+            + Opcode.PUSH4
+            + Opcode.PUSH3
+            + Opcode.PUSH2
+            + Opcode.PUSH1
+            + Opcode.PUSH0
+            + Opcode.PUSH6
+            + Opcode.PACK       # [0, 1, 2, 3, 4, 5]
+            + Opcode.STLOC0
+            + Opcode.LDLOC0
+            + Opcode.PUSH0
+            + check_lower_index_out_of_bounds
+            + Opcode.PUSHM1
+
+            + fix_negative_upper_index
+            + check_upper_index_out_of_bounds
+
+            + Opcode.NEWARRAY0  # starts getting the subarray
+        )
+        expected_possible_negative_index_output = (
+            Opcode.INITSLOT     # with_variables function signature
+            + b'\x01'
+            + b'\x02'
+            + Opcode.PUSH5
+            + Opcode.PUSH4
+            + Opcode.PUSH3
+            + Opcode.PUSH2
+            + Opcode.PUSH1
+            + Opcode.PUSH0
+            + Opcode.PUSH6
+            + Opcode.PACK       # [0, 1, 2, 3, 4, 5]
+            + Opcode.STLOC0
+            + Opcode.LDLOC0
+            + Opcode.LDARG0
+
+            + check_if_fix_negative_index
+            + Integer(5).to_byte_array(min_length=1, signed=True)
+            + fix_negative_lower_index
+            + check_lower_index_out_of_bounds
+
+            + Opcode.LDARG1
+
+            + check_if_fix_negative_index
+            + Integer(6).to_byte_array(min_length=1, signed=True)
+            + fix_negative_upper_index
+            + check_upper_index_out_of_bounds
+
+            + Opcode.NEWARRAY0  # starts getting the subarray
+        )
+
+        path = self.get_contract_path('ListSlicingWithStride.py')
+        output = self.compile(path)
+        self.assertIn(expected_not_negative_index_output, output)
+        self.assertIn(expected_negative_lower_index_output, output)
+        self.assertIn(expected_negative_upper_index_output, output)
+        self.assertIn(expected_possible_negative_index_output, output)
+
+        path, _ = self.get_deploy_file_paths(path)
```

Comment:
don't need to check the opcodes in this case
this test is intended to check the contract behavior only